distribution/docs/src/main/resources/content/_running/ingest-command.adoc

```diff
@@ -33,3 +33,8 @@ This can be the path to a file or a directory containing the desired files.
 ====
 On Windows, put the file path in quotes: `"path/to/file"`.
 ====
+
+Successful command line ingest operations will be accompanied with messaging indicating how many files were ingested and how long the operations took.
+If any files fail to be ingested, the user will be notified with a message indicating how many files failed to be ingested and a recommendation to find information about the failure in the ingest log.
```

Comment:
I think the users will see the message about looking at the ingest log. I don't think we need to tell them about the command's recommendation to look in the log. We could shorten this sentence to:

"The ingest command also prints which files could not be ingested."

Reply:
Reworded